parser/start_tokens.coffee

```diff
@@ -12,7 +12,7 @@ class StartTokens
   @expression = [
     'INTLIT', 'FLOATLIT', 'true', 'false', # Primitive literals
     'ID', '->', 'trait', 'class',          # Misc other tokens
-    @iterable, tokens.reservedWords,
+    @iterable, tokens.beginnings,
```

Comment:
Sure thing! I'll have to do it a little later today.
But everything else in this PR looks good to me.